lib/utils.pm

```diff
@@ -614,9 +614,6 @@ sub reboot_x11 {
                 # Check the password is correct
                 assert_screen 'reboot-auth-correct-password';
             }
-            else {
```

Comment:
Wouldn't it be better to always keep an explicit else branch? What happens if the if does not match?